index.js

```diff
@@ -51,11 +51,11 @@ let dir = argv.dir
 let input = argv._ || argv.input
 let output = argv.output
 
-let options = {
-  parser: argv.parser || false,
-  syntax: argv.syntax || false,
-  strigifier: argv.strigifier || false,
-  map: argv.map || false
+let defaultOptions = {
```

Comment:
Well, no reason for a naming fight here, leave it as is if you think it fits better 😛 , but `defaultOptions` for what at this point? (of course postcss, but the same could be said for defaults), it's not truly obvious from the naming in both ways, only the data-flow adds meaning here later, if so then I prefer the terser one.